pkgs/development/python-modules/parsedmarc/default.nix

```diff
@@ -1,3 +1,24 @@
+{ pythonPackages }:
+let
+  packages = pythonPackages.overrideScope (final: prev: {
```

Comment:
We can't override within `pythonPackages`.
Should we set `meta.broken = true` if the elasticsearch or elasticsearch-dsl version is too high? I assume most of the Python API still works.
We can override the package in `all-packages.nix` or in the NixOS module.

Reply:
Why is that? It works just fine and seems to be done in a few packages already.

Maybe? I don't use it any other way than as a server.

Overriding in `all-packages.nix` feels like a definite anti-pattern, since it means what you're seeing in the package file isn't what you expect. The sane alternative I can think of is to create new packages with the version pinned, but that seemed a bit unnecessary for just one package.

Reply:
> Why is that? It works just fine and seems to be done in a few packages already.

You cannot have two versions of a Python module in the same environment. That could happen if parsedmarc is used together with another module that uses the elasticsearch module.